test/oai/test_mistral.py

```diff
@@ -3,7 +3,16 @@
 import pytest
 
 try:
-    from mistralai.models.chat_completion import ChatMessage
+    from mistralai import (
+        AssistantMessage,
+        Function,
+        FunctionCall,
+        Mistral,
+        SystemMessage,
+        ToolCall,
+        ToolMessage,
+        UserMessage,
+    )
```

Comment:
These are imported but not used?

Reply:
I've just checked, they are all used, not sure why they're coming up as not for you?

![image](https://github.com/user-attachments/assets/e14cdab8-5ba4-430e-a8a7-b31411e9edbb)

However, I did find some others that weren't used, so have cleaned that up.

Reply:
I'll mark as resolved for now.